### PR TITLE
Windows Rust Slave Fixes

### PR DESCRIPTION
**Pull request by jacderida**
Hi Stephen,

This introduces another VM for building Rust code on Windows, the difference being that it removes the dependency on Jenkins, so you don't need a Jenkins master running to spin it up.

It also updates the README to say we need to be using the development version of Ansible for now.

You can run `make travis_rust_slave-windows-2016-x86_64` to bring it up.

Cheers,

Chris

### CODE REVIEW

**Review by S-Coyle — CHANGES_REQUESTED** [review]
Currently `make clean` doesn't destroy the `travis_rust_slave-windows-2016-x86_64` - would be good to ensure it destroys all local VMs

**Comment by S-Coyle** [comment]
@jacderida should this new VM be added to the VM table in the readme?

**Review by S-Coyle — CHANGES_REQUESTED** [review]
Looks like it would make sense to update the readme's Windows Crust example to use the new `travis_rust_slave-windows-2016-x86_64` machine - https://github.com/maidsafe/safe-build-infrastructure/blob/da360cd8345fe17e9bbebbf0bc9c812c1c7c04af/README.md#windows

File: scripts/sh/destroy_local_vms.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+declare -a local_vms=(\
+    "wgclient-ubuntu-bionic-x86_64" \
+    "jenkins_master-centos-7.5-x86_64" \
+    "rust_slave-centos-7.5-x86_64" \
+    "docker_slave-centos-7.5-x86_64" \
+    "rust_slave-ubuntu-trusty-x86_64" \
+    "base-windows-2012_r2-x86_64" \
+    "rust_slave_git_bash-windows-2012_r2-x86_64" \
+    "jenkins_rust_slave-windows-2016-x86_64" \
+    "rust_slave_msys2-windows-2012_r2-x86_64")
```
Comment (S-Coyle) [inline scripts/sh/destroy_local_vms.sh]: Missed `travis_rust_slave-windows-2016-x86_64` from this list

Comment (jacderida) [reply scripts/sh/destroy_local_vms.sh]: Thanks for this. Pushed another commit, and it also contains the documentation change.

**Comment by jacderida** [comment]
> Looks like it would make sense to update the readme's Windows Crust example to use the new `travis_rust_slave-windows-2016-x86_64` machine - https://github.com/maidsafe/safe-build-infrastructure/blob/da360cd8345fe17e9bbebbf0bc9c812c1c7c04af/README.md#windows

I've updated the table with the new machine. I also removed any references to specific products, e.g. "Use for building SCL", as really these machines can be used for building any Rust applications (although sometimes there might be some stuff specific to each product, but those would probably be part of a specific build slave).

I removed the whole section on 'Building Crust', because I don't think it makes much sense to have a reference to Crust here. We should probably just leave the documentation in their own repositories to describe how to build each specific codebase. To me it's fairly obvious that these machines can be used as generic utility machines.

**Review by S-Coyle — APPROVED** [review]

**Comment by S-Coyle** [comment]
Latest changes look good. Approved

**Pull request merged**